src/pages/storage/forms/StoragePoolForm.tsx

```diff
@@ -196,6 +200,7 @@ const StoragePoolForm: FC<Props> = ({ formik, section, setSection }) => {
         active={section}
         setActive={handleSetActive}
         formik={formik}
+        supportedStorageDrivers={supportedStorageDrivers}
```

Comment:
I think it would be simpler to assign a `const isSupportedStorageDriver = supportedStorageDrivers.has(formik.values.driver)` in the component and then forward it here as well as using it for the custom notification below.